lib/pbench/server/api/resources/query_apis/__init__.py

```diff
@@ -235,23 +235,25 @@ def _scroll(self, res_json: JSON) -> JSON:
 
         :return: Postprocessed JSON body to return to client
         """
-        if res_json.get("_scroll_id"):
-            scroll_api_url = f"{self.es_url}/_search/scroll"
-            data = res_json["hits"]["hits"]
-            _scroll_id = res_json["_scroll_id"]
-            while data:
-                scroll_payload = json.dumps({"scroll": "1m", "scroll_id": _scroll_id})
-                scroll_res = requests.post(
-                    scroll_api_url,
-                    data=scroll_payload,
-                    headers={"Content-Type": "application/json"},
-                )
-                scroll_res.raise_for_status()
-                scroll_res = scroll_res.json()
-                data = scroll_res["hits"]["hits"]
-                res_json["hits"]["hits"] += data
-                _scroll_id = scroll_res["_scroll_id"]
-        return res_json
+        if not res_json.get("_scroll_id"):
+            return
+        scroll_api_url = f"{self.es_url}/_search/scroll"
+        data = res_json["hits"]["hits"]
+        scroll_id = res_json["_scroll_id"]
+        while data:
+            scroll_res = requests.post(
+                scroll_api_url,
+                data=json.dumps(
+                    {"scroll": "1m", "scroll_id": scroll_id}
+                ),  # Keeps the scrolling context live for 1 min
+                headers={"Content-Type": "application/json"},
+            )
+            scroll_res.raise_for_status()
+            scroll_res = scroll_res.json()
+            data = scroll_res["hits"]["hits"]
+            res_json["hits"]["hits"] += data
+            scroll_id = scroll_res["_scroll_id"]
+        return
```

Comment:
Since this function doesn't return a value, there's no need to have a `return` statement if it's the last statement in the body.